### DISCUSSION TITLE
Blend opening Windows App SDK project

### DISCUSSION THREAD

**Discussion by BarcodeResource** [microsoft/WindowsAppSDK#1814] [Q&A] [answered]
I am trying to edit a copy of the Style of a XAML element in a Windows App SDK App. When I try to open the project in Blend 2022 in Windows 11 it gives me an error of wapproj project types not supported. Am I missing something or is this not supported yet?

**Comment by andrewleader** [comment, marked as answer]
Hi @BarcodeResource, thanks for asking this. Since this is a XAML-related question, you can ask your question in the XAML repo: https://github.com/microsoft/microsoft-ui-xaml/issues
